<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PendulumWidget</class>
 <widget class="QWidget" name="PendulumWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>384</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>2</horstretch>
    <verstretch>2</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <layout class="QVBoxLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel1">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text">
            <string>&lt;p align=&quot;center&quot;&gt;
m&lt;sub&gt;2&lt;/sub&gt;&lt;br&gt;
-----------&lt;br&gt;
m&lt;sub&gt;1&lt;/sub&gt;+m&lt;sub&gt;2&lt;/sub&gt;
&lt;/p&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_mEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="maxLength">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel2">
           <property name="text">
            <string>&lt;p align=&quot;center&quot;&gt;
l&lt;sub&gt;2&lt;/sub&gt;&lt;br&gt;
------&lt;br&gt;
l&lt;sub&gt;1&lt;/sub&gt;+l&lt;sub&gt;2&lt;/sub&gt;
&lt;/p&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_lEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="maxLength">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel3">
           <property name="text">
            <string>g</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_gEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="maxLength">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel4">
           <property name="text">
            <string>E</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_eEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="maxLength">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel2_2">
           <property name="toolTip">
            <string>specify the time in seconds after which a random perspective change occurs</string>
           </property>
           <property name="text">
            <string>Perspective&lt;br&gt;
Change [s]</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KIntSpinBox" name="m_persSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="m_barColorButton">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>Bars</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_m1ColorButton">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>M1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_m2ColorButton">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>M2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="spacer4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="SsPreviewArea" name="m_preview" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>260</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>SsPreviewArea</class>
   <extends>QWidget</extends>
   <header location="global">sspreviewarea.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">sspreviewarea.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
